src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -110,6 +112,33 @@ const SubmissionDetails = _.flow(
   )(workflows)
 
   const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+
+  // Note: This 'archiveLimitYears' value should reflect the current 'archive-delay' value configured for PROD in firecloud-develop's
+  // 'cromwell.conf.ctmpl' file:
+  const archiveLimitYears = 6
```

Comment:
Based on the tech talk discussion, I think this should be `deletionLimitInYears` since the metadata would still be available on archiving, but not after it is moved to GCS. Same applies for places below: 'archive' => 'deletion' limit